### ISSUE TITLE
Allow for file arborescence

### ISSUE THREAD

**Issue by Popolechien** [openzim/nautilus#43]
Ideally the content should not be a bunch of files but we should have the ability to deal with subfolders

**Comment by rgaudin** [comment]
Well nautilus's spec was “a bunch of files as a ZIM” 😉
I agree this would be a lot better for large collections. This would require a UI do-over which would be welcome as the current one is very basic.

**Comment by Jaifroid** [comment]
I agree this would be nice, but I would just like to point out that no matter how fancy the UI becomes, we shouldn't forget the basics of listing all the final documents in ths ZIMs title index, so that the documents can be found by using native search in a client, and the ZIM can still be accessed by users who cannot run JavaScript (and so cannot run the custom UI). The associated issue is #42.

**Comment by kelson42** [comment]
I'm not super convinced that arborescent view would be the best. Before moving on this I would like to assess this spproach to real problematic use cases.

**Comment by kelson42** [comment]
@Popolechien How would look like the arborescence for these examples.

**Comment by Popolechien** [comment]
Nothing fancy - clicking on a folder icon would open up a new page/list.

**Comment by kelson42** [comment]
How many items? How deeps? Maybe a simple tag system is better!

**Comment by kelson42** [comment]
@Popolechien @rgaudin Although the discussion was somehow suspended to my last question. It seems a decision has been made but this issue has not been updated. I have no strong opinion on this feature request, but would have been great to get this issue following usual discussion and update.

my main concern is that this is an expensive one and worry about the ratio effort/benefit. What is the status here (and in nautilus-webui where I can not find any trace of this feature!)?
